UPGRADE-3.x.md

```diff
@@ -1,6 +1,14 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.25 to 3.26
+=========================
+
+### Commands
+
+Based command class was replace from deprecated `ContainerAwareCommand` to `Command`. Directly access to container will
+no longer be possible. Usage services should be injected instead.
```

Comment:
```suggestion
Command classes were updated to inherit from `Command` instead of deprecated `ContainerAwareCommand`. Directly access to container will
no longer be possible. Services that were retrieved from the DIC MUST be injected instead.
```